python/lsst/analysis/ap/nb_utils.py

```diff
@@ -70,3 +77,222 @@ def make_simbad_link(ra, dec, radius_arcsec=3.0):
         print(f"No matched sources within {radius_arcsec} arcseconds.")
 
         return None
+
+
+def compare_sources(butler1, butler2, query1, query2,
+                    diffimg_type1='goodSeeingDiff',
+                    diffimg_type2='goodSeeingDiff',
+                    badFlagList=None, matchRadius=0.1,
+                    makeCutouts=False, displayCutouts=False,
```

Comment:
Please use snake_case throughout, not a mix: analysis_ap is a new package and has settled on snake_case per [RFC-623](https://jira.lsstcorp.org/browse/RFC-623).